docs/gcp_setup.md

```diff
@@ -0,0 +1,287 @@
+# Google Cloud Platform Setup for a BigQuery Target
+
+## Overview
+
+This page details Google Cloud components required, with recommended minimal privileges, to use GOE in your GCP project.
+
+### Service Account
+
+A service account should be provisioned from the GCP project. This service account can be used by any service that will execute GOE commands, for example it could be attached to a GCE virtual machine.
```

Comment:
Assumes that readers know what GCP, GCE etc stand for (a fair assumption really). But is it worth defining them?

Reply:
Changed, thanks